Phonebook.Frontend/src/app/pages/page-information/page-information.component.scss

```diff
@@ -1,3 +1,28 @@
-.small-card {
-  width: 50%;
+.mat-card{
+  display:flex;
+  flex-direction: column;
 }
+
+// mat-card-footer doesn't align with bottom, you have to user manually styling
+// --> https://github.com/angular/components/issues/11094
+.mat-card-header {
+  flex-shrink: 0;
```

Comment:
same for this

Reply:
> Can you also space it evenly, like this:
> ![image](https://user-images.githubusercontent.com/13590797/67043066-7e835480-f129-11e9-8b42-e35a16011741.png)
> 
> Just add
> 
> ```
> flex-basis: 0;
> flex: 1; 
> ```
> 
> to each card (via css).

We liked the style, that the card are automatically fitting to their content, but if you want so, i change it

Reply:
Oh, i also faulted this in my review just now..

**I would also say that it looks better if the cards all have the same size.**

Reply:
I just thought it looks better... Up to all of us :)